### ISSUE TITLE
Distribute SASS files?

### ISSUE THREAD

**Issue by tgraham-antenna** [docbook/xslTNG#355]
I have no use for this now, and I don't know that I ever will, but is there a future where you would also include the SASS files in the Zip release?

With SASS being more pleasant to work with than its generated CSS, would some people find it easier to make customisations in SASS and then compile their complete CSS rather than tacking their handmade CSS onto the compiled CSS?

It might also get you more coherent PRs than if people report bugs by reference to the compiled CSS.

**Comment by ndw** [comment]
It would be easy enough to do, but would anyone use it? I've been assuming that most users (myself included) just add a new CSS file and override the rules they want to change. But maybe that's because I don't usually make *that* many changes.

**Comment by tgraham-antenna** [comment]
Firstly, there's the question of whether finding the things that you want to change is easier in the SASS or the CSS.

You don't yet use much or any of SASS's extra features (https://sass-lang.com/guide), but it does seem that things like `@include` and `@extend` could make it simpler to, e.g., add new `@page` rules.

I just looked at some changes that I made to the SASS for the Electric Book Works sample, and I do think the changes were simpler than if I'd modified the corresponding CSS.
